docs/Developer's Guide.md

```diff
@@ -35,11 +35,15 @@ Here are the steps to setup the frontend for development:
 
    If it doesn't return a number, you will need to install NodeJS. There are two ways to do this:
 
-   1. Download and run the installer from [the NodeJS Website](https://nodejs.org/en/download/)
+   1. Download and run the installer from [the NodeJS Website](https://nodejs.org/en/download/).  This requires admin (aka "root" or "sudoer") privileges, so if you are on a shared Linx workstation, use one of the options below.
    1. Use a tool that manages Node installations, such as
-      - For any OS: [Fast Node Manager](https://github.com/Schniz/fnm)
-      - For MacOS/Linux: [Node Version Manager](https://github.com/nvm-sh/nvm)
-      - For Windows: [NVM for Windows](https://github.com/coreybutler/nvm-windows)
+      - Node Version Manager (NVM): Install for
+        [MacOS/Linux](https://github.com/nvm-sh/nvm) or
+        [Windows](https://github.com/coreybutler/nvm-windows), then run these
```

Comment:
```suggestion
        [Windows](https://github.com/coreybutler/nvm-windows#installation--upgrades), then run these
```
same as above

Reply:
I put a link to Overview, which mentions an installer, and then goes on to other installation stuff.